Off-Top-App/getUserService
# OT-ContainerizeUserService | Containerize UserService microservice

### What I Did:
1. Used Docker to containerize this microservice
2. Am able to connect this container to it's relative MySQL database container and can successfully make queries and requests to it
3. Connected to the containerized Kafka server and can utilize Kafka within the container

### How To Test:
1. Make sure that you are able to connect to the containerized Kafka/Zookeeper server. The instructions can be found here: https://docs.google.com/document/d/1r5LvXN-08kG8zBy7SFDBJ_80pMHdjJU8RrkNpSqQzAs/edit
2. Follow these instructions: https://docs.google.com/document/d/1yUR_EQBBmefquCQ6KjcEjLAO7YR69vxCo6jR7zqCcn4/edit
3. If you're able to follow these two instructions then this branch is ready to be approved.
